lib/consumer.ex

```diff
@@ -211,7 +211,7 @@ defmodule GenRMQ.Consumer do
   ```
 
   The `reason` argument will either be the atom `:killed` if the Task timed out and needed
-  to be stopped. Or it will be a 2 elementr tuple where the first element is the error stuct
+  to be stopped. Or it will be a 2 element tuple where the first element is the error stuct
```

Comment:
The callback typespec should also be updated here (my bad). I just noticed that the typespec says that `reason` is an `atom()`. It should be an `atom() | {struct(), list()}`

Reply:
cool, will add it then.

Reply:
Done